decidim-core/lib/decidim/core/test/shared_examples/system_endorse_resource_examples.rb

```diff
@@ -79,7 +79,7 @@ def visit_resource
           visit_resource
           within "#resource-#{resource.id}-endorsement-block" do
             click_on "Like"
-            expect(page).to have_button("Dislike")
+            expect(page).to have_no_css('svg use[href*="ri-heart-fill"]')
```

Comment:
This is an a11y smell IMO, as we're only showing the behavior by changing the SVG and not with any other thing to reinforce this. 
How about if we add an `aria-label` with the string "Dislike/Like" and use that in the tests?

Reply:
Actually, i think i do not need to change the label... and should be "Like" / "Dislike".